### ISSUE TITLE
Type 'string' does not satisfy the constraint 'URL'

### ISSUE THREAD

**Issue by pgagnidze** [cloudydeno/deno-notion_sdk#3]
I am getting the following error when running the script with the new stable deno version 1.20.3:
```
Check file:///Users/pgagnidze/Development/personal/coinsearch/src/index.ts
error: TS2344 [ERROR]: Type 'string' does not satisfy the constraint 'URL'.
type _assertSupportedInfoIsSubtype = Assert<RequestInfo, SupportedRequestInfo>
                                                         ~~~~~~~~~~~~~~~~~~~~
    at https://deno.land/x/notion_sdk@v1.0.3/src/fetch-types.ts:9:58
```

You can reproduce it by running the following program https://github.com/pgagnidze/coinsearch/runs/5733563210?check_suite_focus=true

**Comment by danopia** [comment]
Hey, thanks for the report ❤️ 

I think this is related to a supposed-to-not-break change in Deno: https://github.com/denoland/deno/pull/14113 and there is a fix for that breakage https://github.com/denoland/deno/pull/14134

I should be able to put together a fix from my side sometime today, in the meantime going to Deno Canary might help, or the previous stable Deno.

**Comment by danopia** [comment]
I've included a cleanup in v1.0.4 which removes this type brittleness.
